build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.DotNet.Cli.Build
     public class AzurePublisher
     {
         private static readonly string s_dotnetBlobRootUrl = "https://dotnetcli.blob.core.windows.net/dotnet/";
-        private static readonly string s_dotnetBlobContainerName = "dotnet";
+        private static readonly string s_dotnetBlobContainerName = Environment.GetEnvironmentVariable("CONTAINER_NAME");
```

Comment:
I think doing `Environment.GetEnvironmentVariable("CONTAINER_NAME") ?? "dotnet";` might be best here - keep the fallback in case it isn't supplied.

Reply:
Will do